Update Serverless v2 docs with information about default labels (#7314)
kyma-project/kyma: docs/serverless-v2/03-02-api-rules.md

type: Details
---

> By default, lambdas in Kyma are not exposed outside the cluster. To restrict the access, we set automatically the default `serving.knative.dev/visibility=cluster-local` label on KSevice CR, when Function CR is reconciled. Without this label, the Knative Serving controller would create two virtual services, one for the internal cluster communication and the other one for exposing the lambda service outside the cluster. `serving.knative.dev/visibility=cluster-local` label on KSevice CR limits the Knative Serving controller to create only one, local virtual service that allows only for cluster-wide access to lambda services. Such a lambda is accessible for other resources within the cluster under the `{service-name}.{namespace}.svc.cluster.local` endpoint, such as `test-lambda.default.svc.cluster.local`.

Reviewer: ```suggestion
    By default, lambdas in Kyma are not exposed outside the cluster. The Knative Serving controller normally creates two virtual services, one for the internal cluster communication and the other one for exposing the lambda service outside the cluster. To restrict the access, we automatically set the default `serving.knative.dev/visibility=cluster-local` label on a KSevice CR that limits the Knative Serving controller to create only one, local virtual service that allows only for cluster-wide access to lambda services. Such a lambda is accessible for other resources within the cluster under the `{service-name}.{namespace}.svc.cluster.local` endpoint, such as `test-lambda.default.svc.cluster.local`.
```
